sockeye/decoder.py

```diff
@@ -286,23 +282,24 @@ def forward(self, step_input, states):
     def hybrid_forward(self, F, step_input, states):
         mask = None
         if self.inference_only:
-            steps, source_mask, *other = states
+            steps, att_valid_length, *other = states
```

Comment:
Is `att_valid_length` the valid length of source sequence. Maybe we can made it more explicit (e.g., `source_att_valid_length`)

Reply:
good point, renamed